extension-requests/script.js

```diff
@@ -0,0 +1,221 @@
+const container = document.querySelector('.container');
+const extensionRequestsContainer = document.querySelector(
+  '.extension-requests',
+);
+
+const errorHeading = document.querySelector('h2#error');
+const modalParent = document.querySelector('.extension-requests-modal-parent');
+const closeModal = document.querySelectorAll('#close-modal');
+
+//modal containers
+const modalShowInfo = document.querySelector('.extension-requests-info');
+const modalStatusForm = document.querySelector(
+  '.extension-requests-status-form',
+);
+const modalUpdateForm = document.querySelector('.extension-requests-form');
+
+const state = {
+  currentExtensionRequest: null,
+};
+
+const render = async () => {
+  try {
+    addLoader(container);
```

Comment:
[opinion]: utils should be mainpulating main or important resource. Make `addLoader` -> `getLoader`

Reply:
Yeah, I did this as this exact function is defined and used at multiple places so to avoid confusion and consistency, for which I went this way.